data/json/items/comestibles/med.json

```diff
@@ -1610,6 +1698,6 @@
     "addiction_potential": 5,
     "addiction_type": "cocaine",
     "flags": [ "NO_INGEST", "WATER_DISSOLVE" ],
-    "use_action": { "type": "heal", "bleed": 20, "move_cost": 3000, "effects": [ { "id": "pkill1", "duration": 720 } ] }
+    "use_action": { "type": "heal", "bleed": 20, "move_cost": 3000, "effects": [ { "id": "pkill1_generic", "duration": 720 } ] }
```

Comment:
I question if cocaine should have any painkiller effect at all until we can check if you have face injuries for it to be numbing. If it does, it should probably not be extended in duration, unlike all the others... this actually is as long as it lasts as far as I can tell.

Reply:
> I question if cocaine should have any painkiller effect at all until we can check if you have face injuries for it to be numbing. If it does, it should probably not be extended in duration, unlike all the others... this actually is as long as it lasts as far as I can tell.

Insufflated cocaine has about the same full body buzz effect as drinking a beer, which includes mild pain relief. There's definitely evidence for people using stimulants this way: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8791072/

So it shouldn't be a good painkiller but taking the edge off seems appropriate.